src/SDKs/IotHub/Management.IotHub/Microsoft.Azure.Management.IotHub.csproj

```diff
@@ -8,11 +8,11 @@
 		<Description>Provides management capabilities for Microsoft Azure IotHub.</Description>
 		<AssemblyTitle>Microsoft Azure IotHub Management</AssemblyTitle>
 		<AssemblyName>Microsoft.Azure.Management.IotHub</AssemblyName>
-		<Version>2.0.0</Version>
+		<Version>3.0.0</Version>
 		<PackageTags>Microsoft Azure IotHub;IotHub management;IotHub;</PackageTags>
```

Comment:
Went through the PR again and looks like there are no breaking changes introduced here (there were some in the earlier PR though)
In this case, please revert the version to `2.1.0` instead

Reply:
Done